index.html

```diff
@@ -0,0 +1,1610 @@
+                        <p>By <span>Shahed Ghoreishi</span></p>
+                      </div>
+                      
+                        
+                      </div>
+                      
+                    </div>
+
+                    <div class="container-fluid mile">
+                      <div class="media">
+                        <img class="image-fluid mr-3" style="width:70px; height:70px;"
+                                src="https://d.newsweek.com/en/full/1535826/shahed-ghoreishi.png?w=63&h=63&f=2d020d9552d12224d20fbfb1ebe5c3c0"
+                                alt="shaed"
+                              />
+                          <div class="media-body">
+                            <h3><a href="#">With Bolton Gone, It's Time to End Trump's Failed Maximum Pressure Policy</a></h3>
+                            <p>By <span>Shahed Ghoreishi</span></p>
+                          </div>
+                          </div>
+                          
+                        </div>
+
+                        <div class="container-fluid mile">
+                          <div class="media">
+                            <img class="image-fluid mr-3" style="width:70px; height:70px;"
+                                    src="https://d.newsweek.com/en/full/1535826/shahed-ghoreishi.png?w=63&h=63&f=2d020d9552d12224d20fbfb1ebe5c3c0"
+                                    alt="shaed"
+                                  />
+                              <div class="media-body">
+                                <h3><a href="#">With Bolton Gone, It's Time to End Trump's Failed Maximum Pressure Policy</a></h3>
+                                <p>By <span>Shahed Ghoreishi</span></p>
+                              </div>
+                              </div>
+                              
+                            </div>
+
+                            <div class="container-fluid mile">
+                              <div class="media">
+                                <img class="image-fluid mr-3" style="width:70px; height:70px;"
+                                        src="https://d.newsweek.com/en/full/1535826/shahed-ghoreishi.png?w=63&h=63&f=2d020d9552d12224d20fbfb1ebe5c3c0"
+                                        alt="shaed"
+                                      />
+                                  <div class="media-body">
+                                    <h3><a href="#">With Bolton Gone, It's Time to End Trump's Failed Maximum Pressure Policy</a></h3>
+                                    <p>By <span>Shahed Ghoreishi</span></p>
+                                  </div>
+                                  </div>
+                                  
+                                </div>
+
+                                <div class="container-fluid mile">
+                                  <div class="media">
+                                    <img class="image-fluid mr-3" style="width:70px; height:70px;"
```

Comment:
Remove inline CSS from the landing page

Reply:
most of these changes i completed i don't know why they wont show in your browser the link to the live demo https://raw.githack.com/alvinlouis29/Newsweek/features/index.html is here please check it